redux/auth/saga.test.ts

```diff
@@ -194,5 +199,71 @@ describe(`${DOMAIN_NAME}/saga`, () => {
 
       expect(generator.next().done).toBe(true);
     });
+
+    describe('loginWithGoogleSaga', () => {
```

Comment:
This nesting is somehow off :smile: this `describe('loginWIthGoogleSaga', ...)` is inside `describe('loginWithFacebookSaga', ...)`. So you should just move it one level up.

Reply:
Really, how did you found it? :smile: thx!